src/downloader.rs

```diff
@@ -64,6 +64,7 @@ impl Downloader {
 
         Downloader {
             client: reqwest::blocking::ClientBuilder::new()
+                .danger_accept_invalid_certs(true)
```

Comment:
Maybe this should be behind a command line option instead?

Reply:
Yeah I agree